apps/coscrad-frontend/src/components/transcripts/transcript-items-presenter.tsx

```diff
@@ -0,0 +1,35 @@
+import { ITranscriptItem } from '@coscrad/api-interfaces';
+import { Box, Typography } from '@mui/material';
+import { MultilingualTextItemPresenter } from '../../utils/generic-components/presenters/multilingual-text-item-presenter';
+
+interface TranscriptItemsPresenterProps {
+    transcriptItems: ITranscriptItem[];
+}
+
+export const TranscriptItemsPresenter = ({
+    transcriptItems,
+}: TranscriptItemsPresenterProps): JSX.Element => {
+    if (transcriptItems.length === 0) return null;
+
+    return (
+        <>
+            {transcriptItems.map(
+                ({ speakerInitials, inPointMilliseconds, outPointMilliseconds, text }) => (
+                    <Box key={`${speakerInitials}-${inPointMilliseconds}`}>
+                        <Typography component="span" variant="body1" mr={1} fontWeight="bold">
+                            {speakerInitials} [{inPointMilliseconds}-{outPointMilliseconds}]:
+                        </Typography>
+                        {text.items.map(({ text, languageCode, role }) => (
```

Comment:
You need to expose the time stamp info.

Reply:
In the future, you should be able to click a time stamp and scrub to that point in the audio \ video.

Reply:
I'm not sure what you mean by time stamp?  Is that distinct to line 20:

`{speakerInitials} [{inPointMilliseconds}-{outPointMilliseconds}]:`